### DISCUSSION TITLE
申请mysql查询权限无Table级别

### DISCUSSION THREAD

**Discussion by markxu123** [hhyo/Archery#2476] [Q&A] [answered]
申请mysql查询权限时 只能选择DATABASE级别，无TABLE选项，release:1.10.0

**Comment by guocang** [comment]
我也遇到了该问题

**Comment by 397239396** [comment]
一样的问题，有人知道原因吗

**Comment by LeoQuote** [comment, marked as answer]
https://github.com/hhyo/Archery/issues/2289 已经修复了，可以用master分支的代码
